vllm/v1/attention/backends/mla/common.py

```diff
@@ -420,6 +427,12 @@ def __init__(self,
                 dtype=model_config.dtype,
                 device=runner.device,
             )
+            self.workspace = torch.empty(
+                CUDNN_WORKSPACE_SIZE * scheduler_config.max_num_seqs,
+                dtype=torch.int8,
+                device=runner.device,
+            )
```

Comment:
Could we rename this workspace to cudnn_workspace and only create it if VLLM_USE_CUDNN_PREFILL is on?

Also I think we should have a combination check for `envs.VLLM_USE_CUDNN_PREFILL and current_platform.is_device_capability(100)` since the kernel only supports SM100